packages/keyring/src/pair/index.ts

```diff
@@ -98,10 +112,14 @@ function isLocked (secretKey?: Uint8Array): secretKey is undefined {
  * an `encoded` property that is assigned with the encoded public key in hex format, and an `encoding`
  * property that indicates whether the public key value of the `encoded` property is encoded or not.
  */
-export default function createPair ({ toSS58, type }: Setup, { publicKey, secretKey }: PairInfo, meta: KeyringPair$Meta = {}, encoded: Uint8Array | null = null): KeyringPair {
+export default function createPair ({ toSS58, type }: Setup, { publicKey, secretKey }: PairInfo, meta: KeyringPair$Meta = {}, encoded: Uint8Array | null = null, address: Uint8Array | null = null): KeyringPair {
```

Comment:
Not convinced about this addition - since you can take the `publicKey` -> `address` (as per the helper above), why pass it explicitly? It has hoops elsewhere as well, i.e. we are actually passing an empty u8a for ecdsa in the construction.